SupersonicSound/LowLevel/LowLevelSystem.cs

```diff
@@ -240,8 +248,6 @@ private void Dispose(bool disposing)
                 {
                 }
 
-                _system.close().Check();
```

Comment:
See comment above, my suggestion is to add this back in, but with a check on the flag indicating if this is a child of a studio system.